src/store/index.ts

```diff
@@ -17,6 +18,7 @@ export interface CommunityState {
   typedExpressionLanguages: JuntoExpressionReference[];
   groupExpressionRef: string;
   sharedPerspectiveUrl: string;
+  members: Profile[];
```

Comment:
Lets have this as an ad4m Expression type vs Profile so it can evolve as we add more fields to the Profile expression.

Reply:
do you mean the Expression class in the `@perspect3vism/ad4m/Expression` package, because we do get each expression in that format but I am converting it to profile type so that it's easier to consume on the Vue side.

Reply:
Yes exactly. Yeah its definitely useful now but soon we will be using the web components returned from ad4m to render expression data which means we wont need to deserialize the Expression type into our internal type since the returned web component should handle this.

Reply:
If we have a static expression shape inside our store it makes it harder to extend this type later if we update the profile fields

Reply:
@jdeepee I have updated this to use Expression type instead of Profile type.